modules/ppcp-wc-gateway/src/Gateway/PayPalGateway.php

```diff
@@ -543,6 +543,8 @@ function ( PaymentToken $token ): bool {
 				}
 
 				$wc_order->payment_complete();
+				WC()->session->set( 'ppcp_subscription_id', '' );
```

Comment:
Could it be missing somewhere else? Like in `destroy_session_data`.

Maybe this line should be moved there or at least to `handle_payment_success`, `handle_payment_failure`.